xlators/debug/io-stats/src/io-stats.c

```diff
@@ -1398,6 +1404,19 @@ io_stats_dump_global_to_logfp(xlator_t *this, struct ios_global_stats *stats,
                     gf_upcall_list[i], fop_hits, "0", "0", "0");
     }
 
+    /* To measure FOP_HITS_PER_SEC consider time difference
+       between last incremental profile stat started and last
+       fop hit.
+    */
+    duration = (conf->last_fop_hit - stats->started_at);
+    if (duration > 0) {
+        ios_log(this, logfp,
+                "\n"
+                "FOP_HITS_PER_SEC : %" PRIu64,
```

Comment:
"fops/s" may be better IMO. Also, a floating point number could be more interesting given that this won't be an exact average.

Reply:
done